### PR TITLE
Mil 67 api calls

### PR DESCRIPTION
**Pull request by szymonkwak**

### CODE REVIEW

**Comment by matizaj** [comment]
when use 'await' wrap it in try catch block. Try to rewrite then catch block using async await keywords

**Review by matizaj — CHANGES_REQUESTED** [review]
small changes. good work :)

File: frontend/src/utils/getRequest.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import axios from "axios";
+
+const MEDICATION_LIST_ENDPOINT = "https://jsonplaceholder.typicode.com/todos";
+
+const sendGetRequest = (body) => {
```
Comment (CaptainObjective) [inline frontend/src/utils/getRequest.js:5]: I would suggest to create new folder "api" to store code related to api requests

Comment (CaptainObjective) [reply frontend/src/utils/getRequest.js:5]: Also I would suggest to rename function to sth more verbose, e.x: getMedicationsList

**Pull request closed without merging**